sdv/multi_table/hma.py

```diff
@@ -260,12 +260,13 @@ def _extract_parameters(self, parent_row, table_name, foreign_key):
             num_rows = flat_parameters[num_rows_key]
             flat_parameters[num_rows_key] = min(
                 self._max_child_rows[num_rows_key],
-                math.ceil(num_rows)
+                max(0, round(num_rows))
             )
 
         return flat_parameters.rename(new_keys).to_dict()
 
     def _recreate_child_synthesizer(self, child_name, parent_name, parent_row):
+        # HMA only supports one foreign key for each parent/child pair
```

Comment:
I added these comments both to explain why we are only selecting the first foreign key and also so that in the future it's easy to find where the logic needs to be updated to handle multiple foreign keys.

Reply:
I think this comment is misleading. It sounds like if the schema has 2 foreign keys, HMA would crash, which is false.